rapid_pro_tools/rapid_pro_client.py

```diff
@@ -61,6 +61,36 @@ def get_flow_definitions_for_flow_ids(self, flow_ids):
         """
         return self.rapid_pro.get_definitions(flows=flow_ids, dependencies="all")
 
+    def send_message_to_urn(self, message, target_urn):
+        """
+        Sends a message to the given urn.
+
+        :param message: Text of the message to send.
+        :type message: str
+        :param target_urn: URN to send the message to.
+        :type target_urn: str
+        :return: Id of the Rapid Pro broadcast created for this send request.
+        :rtype: int
+        """
+        log.info(f"Sending to '{target_urn}' the message '{message}'...")
```

Comment:
We might want to make this optional, as it will mean that the logs contain quite a lot of PII? I think in some modes we'll certainly want it, but not sure it should be on as info, maybe as verbose?

Reply:
moved to DEBUG, which is the most verbose logging level we have